src/infra/http/controllers/family-member.controller.ts

```diff
@@ -0,0 +1,64 @@
+import { Body, Controller, Delete, Get, Logger, Param, Post, Res } from '@nestjs/common';
+import { ApiBearerAuth, ApiOperation, ApiTags } from '@nestjs/swagger'
+import { Response } from 'express'
+import { ULID } from 'ulidx'
+import { ShowFamilyDoc } from '../decorators/doc/family/show-family-headers.doc'
+import { CreateFamilyMemberUseCase } from 'src/application/use-cases/family-member/create';
+import { DeleteFamilyMemberUseCase } from 'src/application/use-cases/family-member/delete';
+import { ShowFamilyMemberUseCase } from 'src/application/use-cases/family-member/show';
+import { ListFamilyMembersUseCase } from 'src/application/use-cases/family-member/list';
+import { CreateFamilyMemberDto } from '../dtos/family-member/create-family.dto';
+
+@ApiTags('FamilyMember')
+@ApiBearerAuth()
+@Controller('family-members')
```

Comment:
ficaria mais semântico algo do tipo
```suggestion
@Controller('families/:familySerial')
```
pq querendo ou não é uma "extensão" das famílias